Networking/Networking/Model/CreateBlazeCampaign.swift

```diff
@@ -0,0 +1,138 @@
+import Foundation
+import Codegen
+
+/// Represents the entity sent for creating a new Blaze Campaign
+///
+public struct CreateBlazeCampaign: Encodable, Equatable, GeneratedFakeable, GeneratedCopiable {
+    public struct Image: Encodable, Equatable, GeneratedFakeable, GeneratedCopiable {
+        /// URL of the image for the campaign
+        ///
+        public let url: String
+
+        /// mime type of the image
+        ///
+        public let mimeType: String
+
+        public init(url: String, mimeType: String) {
+            self.url = url
+            self.mimeType = mimeType
+        }
+    }
+
+    public struct Targeting: Encodable, Equatable, GeneratedFakeable, GeneratedCopiable {
```

Comment:
We should reuse `BlazeTargetOptions` that was added for fetching forecasted impressions.

Reply:
Good idea. Done in https://github.com/woocommerce/woocommerce-ios/pull/11617/commits/2160b4a7e50956c83266d89c7ac3be9daceb946b